<launch>
  <node pkg="tf" type="static_transform_publisher" name="base_link_to_laser" 
    args="0.0 0.0 0.0 0.0 0.0 0.0 /base_link /laser 40" />

  <node pkg="laserscan" name="laser_odom" type="laser_odom" output="screen">
    <param name="fixed_frame" value = "odom1"/>
    <param name="max_iterations" value="50"/>
    <param name="use_imu" value = "false"/>
    <param name="use_odom" value = "false"/>
    <param name="use_vel" value = "false"/>
    <param name="stamped_vel" value = "false"/>
    <param name="publish_pose_with_covariance" value = "false"/>
    <param name="publish_tf" value = "false"/>
    <param name="use_point_to_line_distance" value = "1"/>
    <param name="epsilon_xy" value = "0.001"/>
    <param name="epsilon_theta" value = "0.001"/>

    <remap from="scan" to="scan_projected"/>
  </node>

  
</launch>
